doc/history.md

```diff
@@ -1,3 +1,6 @@
+## 3.10.3
+* bedrock: add protocol data for 1.19.21 (@stevarino)
```

Comment:
History should be updated in a separate release PR so the GH release action can be triggered